test/ret_test.exs

```diff
@@ -262,6 +264,110 @@ defmodule RetTest do
       assert 0 === count(Asset, target_account)
       assert 0 === count(OwnedFile, target_account)
     end
+
+    test "deletes scenes" do
+      {:ok, admin_account: admin_account} = create_admin_account("admin")
+      target_account = create_account("target")
+
+      project = create_project(target_account)
+      create_scene(project, target_account)
+
+      1 = count(Project, target_account)
+      1 = count(Scene, target_account)
+      5 = count(OwnedFile, target_account)
+
+      assert :ok = Ret.delete_account(admin_account, target_account)
+
+      assert 0 === count(Project, target_account)
+      assert 0 === count(Scene, target_account)
+      assert 0 === count(OwnedFile, target_account)
+    end
+
+    test "deletes parent scenes" do
+      {:ok, admin_account: admin_account} = create_admin_account("admin")
+      target_account = create_account("target")
+      other_account = create_account("other")
+
+      project = create_project(target_account)
+      scene = create_scene(project, target_account)
+      other_project = create_project(other_account)
+      other_project |> Ecto.Changeset.change(parent_scene_id: scene.scene_id) |> Repo.update!()
+      other_scene = create_scene(other_project, other_account)
+      other_scene |> Ecto.Changeset.change(parent_scene_id: scene.scene_id) |> Repo.update!()
```

Comment:
2 tiny nits:
1. Vertical pipelines are much easier to follow than horizontal ones
2. `other_scene` (and potentially `other_project` can be inlined if you aren’t relying on them being explanatory

You could put it all in a single pipeline like so:
```elixir
other_account
|> create_project()
|> Ecto.Changeset.change(parent_scene_id: scene.scene_id)
|> Repo.update!()
|> create_scene(other_account)
|> Ecto.Changeset.change(parent_scene_id: scene.scene_id)
|> Repo.update!()
```

I’m not sure I like two side effects in one pipeline though, so this seems like the happy compromise:
```elixir
other_project =
  other_account
  |> create_project()
  |> Ecto.Changeset.change(parent_scene_id: scene.scene_id)
  |> Repo.update!()

other_project
|> create_scene(other_account)
|> Ecto.Changeset.change(parent_scene_id: scene.scene_id)
|> Repo.update!()
```